Server/JokesOnYou.Web.Api/Controllers/AccountController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using JokesOnYou.Web.Api.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Identity;
+using JokesOnYou.Web.Api.Data;
+using JokesOnYou.Web.Api.Services.Interfaces;
+using JokesOnYou.Web.Api.DTOs;
+using JokesOnYou.Web.Api.Repositories.Interfaces;
+
+namespace JokesOnYou.Web.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
```

Comment:
This api controller does not need [auth] considering its for login/register anyone is able to, instead of adding [allowAnonymous] on every method